PayloadPkg/OsLoader/BootParameters.c

```diff
@@ -9,6 +9,8 @@
 
 CONST CHAR8 *mMmcDllStr = "MMCDLL";
 
+CONST CHAR8 *mAppendParams = NULL;
```

Comment:
This is only for IAS image, right ?
If so, can we make the variable name more specific. EX:  mAppendCmdLineParams